internal/project/model.go

```diff
@@ -1,20 +1,32 @@
 package project
 
 import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/razorpay/metro/internal/auth"
 	"github.com/razorpay/metro/internal/common"
+	"github.com/sethvargo/go-password/password"
 )
 
 const (
 	// Prefix for all project keys in the registry
 	Prefix = "projects/"
+
+	// PartSeparator used in constructing the username parts
+	PartSeparator = "__"
+
+	// perf10__656f81 : sample username format
+	usernameFormat = "%v%v%v"
 )
 
 // Model for a project
 type Model struct {
 	common.BaseModel
-	Name      string            `json:"name"`
-	ProjectID string            `json:"projectId"`
-	Labels    map[string]string `json:"labels"`
+	Name      string                `json:"name"`
+	ProjectID string                `json:"projectId"`
+	Labels    map[string]string     `json:"labels"`
+	AppAuth   map[string]*auth.Auth `json:"app_auth"`
```

Comment:
should we keep a separate model keys under project

projects/{id}/keys

with attributes
- name
- project_id
- secret